mergify_engine/rules/__init__.py

```diff
@@ -375,6 +375,12 @@ def get(self, key: QueueName) -> typing.Optional[QueueRule]:
         except KeyError:
             return None
 
+    def get_prority_level(self, key: QueueName) -> typing.Optional[int]:
+        try:
+            return self[key].config["priority"]
+        except KeyError:
```

Comment:
not a fan because you can have a `KeyError` on `self[key]` or `config["priority"]` and I'm sure the second one would be a bug.
Rather catch the one and only one you want or use `in`.